packages/synapse-react-client/src/components/EntityFinder/tree/VirtualizedTree.tsx

```diff
@@ -77,7 +81,7 @@ function isRootNodeConfiguration(
 
 function hasMoreChildren(node: TreeNode) {
   if (isPaginationNode(node)) {
-    return true
+    return false
```

Comment:
Our "pagination nodes" (empty/placeholders that contain an intersection observer just used to fetch more data) do not have children, so this was confusing. Today, this function is never called with a pagination node, so this shouldn't change anything.